src/org/traccar/protocol/RitiProtocolDecoder.java

```diff
@@ -66,15 +66,15 @@ protected Object decode(
         position.setDeviceId(deviceSession.getDeviceId());
 
         position.set("mode", buf.readUnsignedByte());
-        position.set("command", buf.readUnsignedByte());
-        position.set(Position.KEY_POWER, buf.readUnsignedShort());
+        position.set(Position.KEY_COMMAND, buf.readUnsignedByte());
+        position.set(Position.KEY_POWER, buf.readUnsignedShort());  // mV
```

Comment:
In my review of the code, uses of KEY_BATTERY and KEY_POWER seem to be in units of mV.  Can you confirm?